README.md

```diff
@@ -62,6 +62,10 @@ Assert.MatchSnapshot(snapshotName, objectToSnapshot);
 To update snapshots set the Environment Variable `UpdateSnapshots` to `true` and run the tests.
 You can also add the `[UpdateTestSnapshot]` attribute to your test and run it. (Remember to remove it before you commit your code)
 
+Updates: 
+- UpdateTestSnapshot attribute is renamed to UpdateSnapshots
```

Comment:
Hey @pietgk, thanks for the PR. Would you be okay with updating the line above to just mention the new way of doing things. 
This way the documentation is correct and people don't have to look at the updates section.